content/swift/concepts/classes/classes.md

```diff
@@ -0,0 +1,102 @@
+---
+Title: 'Classes'
+Description: 'Classes serve as a template to define characteristics and behaviors of an object.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Classes'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+Classes serve as a template to define characteristics and behaviors of an object.
+
+They are _reference types_, meaning all _instances_ of a class carry the same characteristics and behaviors.
+
+An instance is an object created based on a class.
+
+## Syntax
+
+In Swift, classes are defined using the `class` keyword, followed by a class name, and a pair of curly braces `{}`.
+
+```swift
+class MyClass {
+  // Class body
+}
+```
+
+**Note:** Class names should be capitalized as per convention.
+
+### `var`, `init()`, and `func`
+
+Variables can be defined using the keyword `var`, followed by the variable name, and a value assignment.
+
+`init()` is a special method that gets called when an instance is created. It can take any number of arguments in the form of `property: DataType`. The `self` keyword refers to the instance itself.
```

Comment:
```suggestion
`init()` is a special method that gets called when a class instance is created. It can take any number of arguments in the form of `property: DataType`. The `self` keyword refers to the instance itself.
```
```suggestion
`init()` is a special method that gets called when an instance is created. It can take any number of arguments in the form of `property: DataType`. The `self` keyword refers to the instance itself.
```